### PR TITLE
Release 13.3.5

### PR DESCRIPTION
**Pull request by camscale**
* Fixed a bug in teleport-cluster Helm chart causing Teleport to crash when AWS DynamoDB autoscaling is enabled. [#30841](https://github.com/gravitational/teleport/pull/30841)
* Added Teleport Assist to Web Terminal. [#30811](https://github.com/gravitational/teleport/pull/30811)
* Fixed S3 metric name for completed multipart uploads. [#30710](https://github.com/gravitational/teleport/pull/30710)
* Added the ability for `tsh` to register and enroll the `--current-device`. [#30702](https://github.com/gravitational/teleport/pull/30702)
* Fixed Review Requests to disallow reviews after request is resolved. [#30690](https://github.com/gravitational/teleport/pull/30690)
* Ensure that SSH session errors are reported to the terminal. [#30684](https://github.com/gravitational/teleport/pull/30684)
* Fixed an issue with `tsh aws ssm start-session`. [#30668](https://github.com/gravitational/teleport/pull/30668)
* Fixed an issue with the access request failing with `invalid maxDuration`. [teleport.e#2037](https://github.com/gravitational/teleport.e/pull/2037)

### Security fix

* Security improvements with possible `medium` severity DoS conditions through protocol level attacks. [#30854](https://github.com/gravitational/teleport/pull/30854)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 13.3.5 (08/22/23)
+
+* Fixed S3 metric name for completed multipart uploads. [#30710](https://github.com/gravitational/teleport/pull/30710)
+* Added `--current-device` capabilities to `tsh`. [#30702](https://github.com/gravitational/teleport/pull/30702)
```
Comment (zmb3) [inline CHANGELOG.md:6]:
```suggestion
* Added the ability for `tsh` to register and enroll the `--current-device`. [#30702](https://github.com/gravitational/teleport/pull/30702)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+* Fixed Review Requests to disallow reviews after request is resolved. [#30690](https://github.com/gravitational/teleport/pull/30690)
+* Fixed Teleport Connect to properly show errors from the remote end. [#30684](https://github.com/gravitational/teleport/pull/30684)
+* Fixed an issue with `tsh aws ssm start-session`. [#30668](https://github.com/gravitational/teleport/pull/30668)
+
```
Comment (jakule) [inline CHANGELOG.md:10]:
```suggestion
* Fix the issue with the access request failing with `invalid maxDuration`. [https://github.com/gravitational/teleport.e/pull/2037](https://github.com/gravitational/teleport.e/pull/2037)
```

**Review by r0mant — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 13.3.5 (08/22/23)
+
+* Fixed S3 metric name for completed multipart uploads. [#30710](https://github.com/gravitational/teleport/pull/30710)
+* Added `--current-device` capabilities to `tsh`. [#30702](https://github.com/gravitational/teleport/pull/30702)
+* Fixed Review Requests to disallow reviews after request is resolved. [#30690](https://github.com/gravitational/teleport/pull/30690)
+* Fixed Teleport Connect to properly show errors from the remote end. [#30684](https://github.com/gravitational/teleport/pull/30684)
+* Fixed an issue with `tsh aws ssm start-session`. [#30668](https://github.com/gravitational/teleport/pull/30668)
```
[the remark this answers is not in the document]
Comment (greedy52) [reply CHANGELOG.md:9]: It didn't work before. Maybe:
- Fixed an issue `tsh aws ssm start-session` throws an error

**Comment by zmb3** [comment]
@jentfoo did you purposely push changes to this branch? In most cases, the release commit only updates versions and the changelog. Is there any reason your fuzzing changes can't be backported in the typical manner?

**Comment by jentfoo** [comment]
@zmb3 It's a late inclusion, from discussions with @camscale and @r0mant we thought this would be easiest

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
+### Security fix
+
+* TODO(jent): Update this when security fix lands
```
Comment (jentfoo) [inline CHANGELOG.md:13]:
```suggestion
* Improvements with possible `medium` severity DoS conditions through protocol level attacks.
```

File: CHANGELOG.md (hunk 1 above)
Comment (jakule) [inline CHANGELOG.md:6]: We also merged new Assist features to v13 today:
```suggestion
* Added `--current-device` capabilities to `tsh`. [#30702](https://github.com/gravitational/teleport/pull/30702)
* Added Teleport Assist to Web Terminal. [#30811](https://github.com/gravitational/teleport/pull/30811)
```

File: CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (camscale) [reply CHANGELOG.md:9]: I tried to figure out a better description for this one as I didn't know how to describe "endpoint URL mode" concisely and usefully.

**Review by jentfoo — APPROVED** [review]

**Pull request merged**